tests/integration/test_yarn.py

```diff
@@ -48,6 +48,17 @@
             ),
             id="yarn_correct_version_installed_by_corepack",
         ),
+        pytest.param(
+            utils.TestParameters(
+                repo="https://github.com/cachito-testing/cachi2-yarn-berry.git",
+                ref="53a2bfe8d5ee7ed9c2f752fe75831a881d54895f",
+                packages=({"path": ".", "type": "yarn"},),
+                check_vendor_checksums=False,
+                expected_exit_code=0,
+                expected_output="Processing the request using yarn@4.5.2",
+            ),
+            id="yarn_v4",
+        ),
```

Comment:
Shouldn't this be among e2e tests with a Dockerfile?

Reply:
My reasoning that a more simple integration test would cover the new use case. Do you think we need to have a full e2e test to it? Any special reason for it?